client/my-sites/themes/themes-magic-search-card/taxonomy-styling.js

```diff
@@ -0,0 +1,36 @@
+/**
+	* This is a helper file that establishes designed visual connection between
+	* search taxonomy and its graphical representation.
+	*/
+
```

Comment:
Since this is styling, I think it should rather be done in the `.scss` files. This means (at least for the colors), you add a 

```js
`themes-magic-search-card__welcome-taxonomy-${ taxonomy }
```

(or so) `className`, and add styling for each individual one to the `.sccs`.